docker-images/kafka/cruise-control-scripts/cruise_control_config_generator.sh

```diff
@@ -43,12 +43,16 @@ cluster.configs.file=$CC_CLUSTER_CONFIG_FILE
 webserver.accesslog.path=$CC_ACCESS_LOG
 webserver.http.address=0.0.0.0
 webserver.http.cors.allowmethods=OPTIONS,GET
+webserver.ssl.keystore.location=/tmp/cruise-control/client-server.keystore.p12
+webserver.ssl.keystore.password=$CERTS_STORE_PASSWORD
+webserver.ssl.keystore.type=PKCS12
+webserver.ssl.key.password=$CERTS_STORE_PASSWORD
 security.protocol=SSL
 ssl.keystore.type=PKCS12
-ssl.keystore.location=/tmp/cruise-control/replication.keystore.p12
+ssl.keystore.location=/tmp/cruise-control/client-server.keystore.p12
 ssl.keystore.password=$CERTS_STORE_PASSWORD
 ssl.truststore.type=PKCS12
-ssl.truststore.location=/tmp/cruise-control/replication.truststore.p12
+ssl.truststore.location=/tmp/cruise-control/client.truststore.p12
```

Comment:
Again, the naming seems to be a bit unintuitice ... should this be called something like `kafka.truststore` or something? Or keep the old name whicih was much more clear than this.

Reply:
I like your suggestion for the keystore:  `cruise-control.keystore.p12` and  I was thinking we could use the same pattern for the truststore: `cruise-control.trustore.p12` since it is cruise control's truststore. Although I do understand the reasoning behind the original naming scheme that the truststore contains the public Kafka certs. What do you think?

Reply:
`cruise-control.trustore.p12` reads as _This is the truststore used for mTLS authentication of users connecting to the Cruise Control APIs._. But it is not that, or? It is the _Truststore used to trust the Kafka brokers when connecting to them_. So if I got it right, I think something what relates it to the Kafka cluster instead of Cruise Control makes more sense.

Reply:
I was reading `cruise-control.truststore.p12` as _This is the truststore holding the certs used for identifying apps which Cruise Control trusts_. From what I understood a truststore can hold multiple certificate entries used for identifying different apps ( although in this specific case, Cruise Control only needs the certs for identifying/trusting the Kafka brokers).

Reply:
Well, I guess this _This is the truststore holding the certs used for identifying apps which Cruise Control trusts_ applies for both cases ... but at least me personally, I would by the name link it to the Cruise Control API which for me is the main CC interface. Not to the Kafka client which is inside.

Reply:
I understand, in that case I will switch it back to what is was originally, `replication.truststore.p12`, I want it to be as clear as possible and there is no sense in me changing the name if it is going to mislead people!